scripts/openmc-make-depletion-chain

```diff
@@ -15,15 +15,29 @@ URLS = [
 ]
 
 def main():
-    for url in URLS:
-        basename = download(url)
-        with ZipFile(basename, 'r') as zf:
-            print('Extracting {}...'.format(basename))
-            zf.extractall()
-
-    decay_files = glob.glob(os.path.join('decay', '*.endf'))
-    nfy_files = glob.glob(os.path.join('nfy', '*.endf'))
-    neutron_files = glob.glob(os.path.join('neutrons', '*.endf'))
+    endf_dir = os.environ.get("OPENMC_ENDF_DATA")
+    if endf_dir is not None:
+        endf_dir = Path(endf_dir)
+    elif all(os.path.isdir(lib) for lib in ("neutrons", "decay", "nfy")):
+        endf_dir = Path(".")
+    else:
+        for url in URLS:
+            basename = download(url)
+            with ZipFile(basename, 'r') as zf:
+                print('Extracting {}...'.format(basename))
+                zf.extractall()
+        endf_dir = Path(".")
+
+    decay_files = tuple((endf_dir / "decay").glob("*endf"))
+    neutron_files = tuple((endf_dir / "neutrons").glob("*endf"))
+    nfy_files = tuple((endf_dir / "nfy").glob("*endf"))
+
+    # check files exist
+    for flist, ftype in zip(
+            (decay_files, neutron_files, nfy_files),
+            ("decay", "neutron", "nfy")):
```

Comment:
`zip` seems unnecessary here. Why not just make it `[(decay_files, "decay"), ...]`?

Reply:
No strong reason. Easy fix